examples/examples/map-traced-error.rs

```diff
@@ -0,0 +1,82 @@
+//! An example on composing errors inside of a `TracedError`, such that the
+//! SpanTrace captured is captured when creating the inner error, but still wraps
+//! the outer error.
+#![deny(rust_2018_idioms)]
+#![allow(clippy::try_err)]
+use std::error::Error;
+use std::fmt;
+use tracing_error::{prelude::*, ErrorLayer, TracedError};
```

Comment:
It looks like the example needs to be updated to track that `ErrorLayer` was renamed on master:
```suggestion
use tracing_error::{prelude::*, ErrorSubscriber, TracedError};
```